### ISSUE TITLE
login not detected at forum.xda-developers.com

### ISSUE THREAD

**Issue by Strubbl** [nextcloud/passman-webextension#205]
### Steps to reproduce
1. go to https://forum.xda-developers.com
2. login

### Expected behaviour
Web extension should give popup for detecting new account.

### Actual behaviour
Nothing happens. 

### Configuration
**Operating system**: Archlinux

**Browser**: Firefox 56

**Extensions that might cause interference**: none

**Passman version**:  2.1.4

**Extension version**: 2.0.4 

**Nextcloud version**: 12.0.3
   
#### Browser log
<details>
<summary>Browser log</summary>
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by brantje** [comment]
![image](https://user-images.githubusercontent.com/2758767/32414546-f1e18220-c229-11e7-86e4-aca026157e84.png)
From what page do you login? The main page?

**Comment by brantje** [comment]
I tried with disabled adblocker, then it works fine.

**Comment by brantje** [comment]
nvm, that error appears on other pages too. Need to dig deeper.

**Comment by Strubbl** [comment]
Yes, i use ublock origin and get the error `Uncaught ReferenceError: Criteo is not defined`, too.

**Comment by duongpro01** [comment]
hummm, that error appears on other pages too.

**Comment by binsky08** [comment]
This project has been completely rewritten (v3).
The old code is archived in the `legacy-v2` branch.
Please check whether the issue still exists in the current v3, and if so, please report it again.
